doc/releases/release-notes-3.7.rst

```diff
@@ -634,11 +657,16 @@ Drivers and Sensors
 
 * MIPI-DBI
 
+    * Added release API
+    * Added support for mode selection via the device tree
+
 * Pin control
 
 * PWM
 
   * Added support for STM32H7R/S series.
+    * Added a Add QTMR PWM driver for NXP imxrt11xx
```

Comment:
It would probably be faster if you build the documentation locally to be sure it renders properly.
Otherwise you can check the result of the rendering done in CI for the PR https://builds.zephyrproject.io/zephyr/pr/75274/docs/releases/release-notes-3.7.html

Reply:
> It would probably be faster if you built the documentation locally to be sure it renders properly. Otherwise you can check the result of the rendering done in CI for the PR https://builds.zephyrproject.io/zephyr/pr/75274/docs/releases/release-notes-3.7.html (that would be a very long cycle, but much faster than waiting for reviewers to spot things)

Well, I did build locally, the build process failed because of processes crashing as does check compliance for reasons that have nothing to do with my change (Kconfig related). I am not one to rely on other people's time "to spot things". Sorry if it appears that way.